vlasiator.cpp

```diff
@@ -137,11 +137,11 @@ void computeNewTimeStep(dccrg::Dccrg<SpatialCell,dccrg::Cartesian_Geometry>& mpi
                    blockParams[blockLID * BlockParams::N_VELOCITY_BLOCK_PARAMS + BlockParams::VZCRD] +
                    (i + HALF) * blockParams[blockLID * BlockParams::N_VELOCITY_BLOCK_PARAMS + BlockParams::DVZ] + EPS;
 
-               const Real dt_max_cell = min(dx / fabs(Vx), min(dy / fabs(Vy), dz / fabs(Vz)));
-               cell->parameters[CellParams::MAXRDT] = min(dt_max_cell, cell->parameters[CellParams::MAXRDT]);
+               const Real dt_max_cell = min({dx / fabs(Vx), dy / fabs(Vy), dz / fabs(Vz)});
                cell->set_max_r_dt(popID, min(dt_max_cell, cell->get_max_r_dt(popID)));
             }
          }
+         cell->parameters[CellParams::MAXRDT] = min(cell->get_max_r_dt(popID), cell->parameters[CellParams::MAXRDT]);
```

Comment:
still needs to be inside the pop loop!

Reply:
I think it is, otherwise it won't compile?

Reply:
Sorry, my bad :) For some reason even looking at it multiple times, I didn't see the i loop until going out of diff mode.